rules/otbn.bzl

```diff
@@ -148,6 +148,45 @@ def _otbn_binary(ctx):
         ),
     ]
 
+def _otbn_test(ctx):
```

Comment:
Maybe the name should reflect the fact that this is running in the simulator (which itself is somewhat ambiguously named, and should maybe be called an interpreter)... so something like `otbn_sim_test`?

Reply:
Done!